libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationNotificationProvider.java

```diff
@@ -8,6 +8,7 @@
 class NavigationNotificationProvider {
 
   private NavigationNotification navigationNotification;
+  private boolean isUpdating = true;
```

Comment:
NIT - what about `shouldUpdate` instead? It sounds better to me. By default, `shouldUpdate`? Yes - `true` After shutting down, `shouldUpdate`? No - `false`
What do you think?